tensorpipe/config.h.in

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#cmakedefine01 TENSORPIPE_HAS_CUDA
```

Comment:
This macro must be public because users need to check it in order to see whether the pipe support CudaTensors, right? For the same reason as above, should we call it `TENSORPIPE_SUPPORTS_CUDA_TENSORS` or something in that vein, to be more explicit?

Reply:
Maybe this one we can call `TENSORPIPE_SUPPORTS_CUDA`, to avoid the `HAS` verb which is the same we use for transports/channels and which may trick users into thinking that this variable says whether there exists a channel called "cuda"?